src/legacy/core_plugins/kibana/public/management/app.html

```diff
@@ -1,4 +1,4 @@
-<div class="app-container euiPage">
+<div class="euiPage mgtPage">
   <div id="management-sidenav" class="euiPageSideBar" style="position: static;"></div>
-  <main class="management-container euiPageBody euiPageBody--restrictWidth-default" ng-transclude></main>
+  <main class="euiPageBody euiPageBody--restrictWidth-default mgtPageBody" ng-transclude></main>
```

Comment:
Is `mgtPageBody` supposed to be `mgtPage__body`?

Reply:
Yes, this fixed the issue @cjcenizal noticed.